rocky/tests/test_organization.py

```diff
@@ -130,4 +131,32 @@ def test_organization_member_give_and_revoke_clearance_no_action_reloads_page(
     assertContains(response, "Add new member")
     assertContains(response, my_user.email)
     assertContains(response, "Grant")
-    assertContains(response, "active")
+
+
+def test_organization_does_not_exist(client, normal_user, organization):
+    client.force_login(normal_user)
+    response = client.get(reverse("organization_detail", kwargs={"organization_code": "nonexistent"}))
+
+    assert response.status_code == 404
+
+
+def test_organization_no_member(client, normal_user_without_organization_member, organization):
+    client.force_login(normal_user_without_organization_member)
+    response = client.get(reverse("organization_detail", kwargs={"organization_code": organization.code}))
+
+    assert response.status_code == 404
+
+
+def test_organization_active_member(rf, normal_user, organization):
+    request = setup_request(rf.get("organization_detail"), normal_user)
+    response = OrganizationDetailView.as_view()(request, organization_code=organization.code)
+
+    assert response.status_code == 200
```

Comment:
No checks for page content? (Not sure if this is the relevant one, but it would be nice to have a test proving we really see the correct member list on the member overview page given the filters?)

Reply:
Done